packages/next-app/pages/[kanbanId]/index.tsx

```diff
@@ -168,3 +176,29 @@ function NewTask({ task, isPM, kanban, address }) {
     </div>
   );
 }
+
+function ProgressTask({ task, kanban, }) {
+  const { id, title, detail, requests, funds,raider } = task;
+  const [loading, setLoading] = useState(false);
+  
+  const onSubmitTask = async () => {
+    const kb = kanban as Kanban;
+    const tasklog = await kb.taskLog(id); 
+    tasklog.reviewed;
```

Comment:
Not sure what your trying to do here. Call the contract method here instead (kb.taskForReview(id))